Makefile

```diff
@@ -43,8 +43,14 @@ CTR_CMD_PUSH_OPTIONS ?=
 
 GENERAL_TAGS := 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo libbpf '
 GPU_TAGS := ' gpu '
+ifeq ($(shell ldconfig -p | grep -q libnvml_injection.so && echo exists),exists)
+	GPU_TAGS := ' nvml '
```

Comment:
```suggestion
	GPU_TAGS := "$(GPU_TAGS) nvml"
```

Reply:
IMHO this should append to the list - so you could build with all things supported if your system has all libraries available.

Reply:
I think it's unlikely to have a system that has Habana + Nvidia GPUs. And TBH if the system has both NVML  libraries and DCGM we prioritise the use of dcgm. so rather than building aggregating the tag I had just set it flat out.

Reply:
From an image POV atm we have separate images also:
![Screenshot 2024-06-19 at 14 54 19](https://github.com/sustainable-computing-io/kepler/assets/5012288/651b4f07-b28a-45d7-8030-def5ea5eb72d)

For these reasons I'm less inclined to aggregate the tag (ATM). If we decide to change the image builds then absolutely